conans/test/conftest.py

```diff
@@ -23,6 +23,12 @@
         "3.16": {"disabled": True},
         "3.17": {"disabled": True},
         "3.19": {"path": {"Windows": "C:/ws/cmake/cmake-3.19.7-win64-x64/bin"}},
+        # To explicitly skip one tool for one version, define the path as None
```

Comment:
I think I would reverse the logic, and make it explicit: ``"Darwin": "skip-tests"``, because the semantics are not well defined, what happens with Linux in this case, it is None too, so it will be skipped? Doing this, the ``undefined`` in the code below is not necessary, ``None`` becomes undefined (pythonic), and ``skip-tests`` means skip those tests. Wdyt?

Reply:
Yes, it's more to reverse the logic and use the `skip-tests`, I saw that we had this previously and thought that maybe we were using this to explicitly skip: https://github.com/conan-io/conan/blob/260aedbaa49dea43711b38bbdc0fdbf6d2c65dad/conans/test/conftest.py#L85-L86 so I maintained that in case it was used in the confuser, but I will change that for a `skip-tests` there and do the changes 👍